### PR TITLE
Ft comment endpoints 165412922

### PR DESCRIPTION
**Pull request by MBenedicte**
#### What does this PR do?
Add endpoints for comments (CRUD)

#### Description of Task to be completed?
Having the following endpoints:  
GET /API/V1/articles  
GET /API/V1/articles/:articleId  
GET /api/v1/articles/:articleId/comments  
POST /api/v1/articles/:articleId/comments  
PUT /api/v1/articles/:articleId/comments/:commentId  
EDIT /api/v1/articles/:articleId/comments/:commentId

#### How should this be manually tested?
Using postman to test the endpoints

#### Any background context you want to provide?
Developed using Nodejs and express

#### What are the relevant pivotal tracker stories?
165412922

### CODE REVIEW

File: src/tests/middlewares/checkComment.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+delete newArticle.id;
+delete newComment.id;
+
+describe('COMMENTS', () => {
+  it('Should not let the user continue with wrong input', (done) => {
+    newComment.userId = createdUser.id;
+    chai
+      .request(app)
+      .post('/api/v1/articles/111111111/comments')
+      .send('~~~~~')
```
Comment (e-liyai) [inline src/tests/middlewares/checkComment.js]: @MBenedicte what's this line meant to do?

File: src/tests/routes/comments.test.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+      await db.Comment.destroy({
+        truncate: true,
+        cascade: true,
+        logging: false
+      });
+      createdUser = (await db.User.create(newUser, { logging: false })).dataValues;
+      newArticle.userId = createdUser.id;
+      createdArticle = (await db.Article.create(newArticle, { logging: false })).dataValues;
+      newComment.articleId = newArticle.id;
+      // createdComment = (await db.Comment.create(newComment, { logging: false })).dataValues;
```
Comment (e-liyai) [inline src/tests/routes/comments.test.js]: @MBenedicte delete the comment if not in use

File: src/routes/api/comments.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { Router } from 'express';
+import CommentController from '../../controllers/CommentController';
+import Validation from '../../middlewares/validateComment';
+import checkArticle from '../../middlewares/checkArticle';
+import checkComment from '../../middlewares/checkComment';
+import errorHandler from '../../middlewares/errorHandler';
+
+const router = Router();
+
+// router.post('/:articleId/comments', Validation, checkArticle, CommentController.create);
```
Comment (e-liyai) [inline src/routes/api/comments.js]: @MBenedicte delete this commented out code

File: src/tests/middlewares/checkComment.js (hunk 1 above)
Comment (MBenedicte) [reply src/tests/middlewares/checkComment.js]: It is to test a bad input from the user..

File: src/tests/routes/comments.test.js (hunk 2 above)
Comment (MBenedicte) [reply src/tests/routes/comments.test.js]: Yes.

File: src/routes/api/comments.js (hunk 3 above)
Comment (MBenedicte) [reply src/routes/api/comments.js]: YEAH

**Pull request merged**